components/formats-gpl/src/loci/formats/in/ZeissLSMReader.java

```diff
@@ -2148,9 +2148,9 @@ private Object readValue() throws IOException {
 
     switch (blockType) {
       case TYPE_LONG:
-        return new Long(in.readInt());
+        return (long) in.readInt();
```

Comment:
I think you're correct. It'd mean we have a `Integer` instead of a `Long` in `blockData` but I don't think that would surprise other code. Let's try it and see.